subprojects/gradle/module-types/src/main/kotlin/com/avito/android/module_type/DependencyRestriction.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.android.module_type
+
+import com.avito.module.configurations.ConfigurationType
+
+public class DependencyRestriction(
+    public val matcher: DependencyMatcher,
+    public val exclusions: Set<DependencyMatcher> = emptySet(),
+    public val description: String
+)
+
+internal fun DependencyRestriction.isViolated(
```

Comment:
it could be replaced with simple internal fun

Reply:
Ok, will move it